cvpy/biomedimage/BiomedImage.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+''' BioMedImage analysis tools '''
+
+import sys
+import struct
+import numpy as np
+from typing import List
+from warnings import warn
+from swat import CAS, CASTable
+
+# from cvpy.biomedimage.LabelConnectivity import LabelConnectivity
+
+class BiomedImage(object):
+    '''
+    This is the biomedimage class, with several biomedimage-related functions implemented under it. 
+    '''
+
+    def __init__(self, cas_session: CAS = None) -> None:
+        '''
+        Constructor for project class
+        :param cas_session: the CAS session for this project
+        '''
+        self._cas_session = cas_session
+    
+    @property
+    def cas_session(self) -> CAS:
+        return self._cas_session
+
+    @cas_session.setter
+    def cas_session(self, cas_session) -> None:
+        self._cas_session = cas_session
+
+
+    def quantify_sphericity(self, image_table: CASTable, use_spacing: bool, input_background: float,
```

Comment:
We talked about creating an input table object that different functions will have as an argument (as opposed to directly using the SWAT CASTable object). Did we decide to do that later on? I can't remember. If we're doing that later on, can you go ahead and create a story to do that and put it in a future sprint?

Reply:
@fijoy, @mumuMia - I created an ImageTable object in my PR. 

https://github.com/sassoftware/python-cvpy/pull/14